### ISSUE TITLE
AttributeError: 'VistirSpinner' object has no attribute '_color_func' (2022.11.4)

### ISSUE THREAD

**Issue by LukeNotable** [pypa/pipenv#5458]
### Issue description

Suddenly with 2022.11.4, running `pipenv install` even on an empty directory (under Python 3.9):
```
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/dist-packages/pipenv/utils/spinner.py", line 18, in create_spinner
    yield sp
  File "/usr/local/lib/python3.9/dist-packages/pipenv/core.py", line 1019, in do_create_virtualenv
    sp.green.ok(
  File "/usr/local/lib/python3.9/dist-packages/pipenv/vendor/vistir/spin.py", line 264, in ok
    self._freeze(_text, err=err)
  File "/usr/local/lib/python3.9/dist-packages/pipenv/vendor/vistir/spin.py", line 363, in _freeze
    last_frame = self._compose_out(text, mode="last")
  File "/usr/local/lib/python3.9/dist-packages/pipenv/vendor/vistir/spin.py", line 386, in _compose_out
    if self._color_func is not None:
  File "/usr/local/lib/python3.9/dist-packages/pipenv/vendor/vistir/spin.py", line 115, in __getattr__
    retval = super(DummySpinner, self).__getattribute__(k)
AttributeError: 'VistirSpinner' object has no attribute '_color_func'

Traceback (most recent call last):
  File "/usr/local/bin/pipenv", line 8, in <module>
    sys.exit(cli())
  File "/usr/local/lib/python3.9/dist-packages/pipenv/vendor/click/core.py", line 1128, in __call__
    return self.main(*args, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/pipenv/cli/options.py", line 57, in main
    return super().main(*args, **kwargs, windows_expand_args=False)
  File "/usr/local/lib/python3.9/dist-packages/pipenv/vendor/click/core.py", line 1053, in main
    rv = self.invoke(ctx)
  File "/usr/local/lib/python3.9/dist-packages/pipenv/vendor/click/core.py", line 1659, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/usr/local/lib/python3.9/dist-packages/pipenv/vendor/click/core.py", line 1395, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/usr/local/lib/python3.9/dist-packages/pipenv/vendor/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/pipenv/vendor/click/decorators.py", line 84, in new_func
    return ctx.invoke(f, obj, *args, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/pipenv/vendor/click/core.py", line 754, in invoke
    return __callback(*args, **kwargs)
  File "/usr/local/lib/python3.9/dist-packages/pipenv/cli/command.py", line 236, in install
    do_install(
  File "/usr/local/lib/python3.9/dist-packages/pipenv/core.py", line 2115, in do_install
    ensure_project(
  File "/usr/local/lib/python3.9/dist-packages/pipenv/core.py", line 530, in ensure_project
    ensure_virtualenv(
  File "/usr/local/lib/python3.9/dist-packages/pipenv/core.py", line 463, in ensure_virtualenv
    do_create_virtualenv(
  File "/usr/local/lib/python3.9/dist-packages/pipenv/core.py", line 1019, in do_create_virtualenv
    sp.green.ok(
  File "/usr/local/lib/python3.9/dist-packages/pipenv/vendor/vistir/spin.py", line 264, in ok
    self._freeze(_text, err=err)
  File "/usr/local/lib/python3.9/dist-packages/pipenv/vendor/vistir/spin.py", line 363, in _freeze
    last_frame = self._compose_out(text, mode="last")
  File "/usr/local/lib/python3.9/dist-packages/pipenv/vendor/vistir/spin.py", line 386, in _compose_out
    if self._color_func is not None:
  File "/usr/local/lib/python3.9/dist-packages/pipenv/vendor/vistir/spin.py", line 115, in __getattr__
    retval = super(DummySpinner, self).__getattribute__(k)
AttributeError: 'VistirSpinner' object has no attribute '_color_func'
```

Nothing additional from verbose.

-------------------------------------------------------------------------------

<details><summary>$ pipenv --support</summary>

Pipenv version: `'2022.11.4'`

Pipenv location: `'/usr/local/lib/python3.9/dist-packages/pipenv'`

Python location: `'/usr/bin/python3.9'`

OS Name: `'posix'`

User pip version: `'22.3'`

user Python installations found:

  - `3.9.15`: `/usr/bin/python3.9`
  - `3.6.9`: `/usr/bin/python3.6`
  - `3.6.9`: `/usr/bin/python3.6m`
  - `3.6.9`: `/usr/bin/python3`

PEP 508 Information:

```
{'implementation_name': 'cpython',
 'implementation_version': '3.9.15',
 'os_name': 'posix',
 'platform_machine': 'x86_64',
 'platform_python_implementation': 'CPython',
 'platform_release': '5.15.49-linuxkit',
 'platform_system': 'Linux',
 'platform_version': '#1 SMP PREEMPT Tue Sep 13 07:51:32 UTC 2022',
 'python_full_version': '3.9.15',
 'python_version': '3.9',
 'sys_platform': 'linux'}
```

System environment variables:

  - `OLDPWD`
  - `_`
  - `PATH`
  - `CERTIFICATES_ARG`
  - `SHLVL`
  - `TERM`
  - `DEBIAN_FRONTEND`
  - `HOME`
  - `PWD`
  - `DISPLAY`
  - `LANG`
  - `LS_COLORS`
  - `LC_ALL`
  - `GRPC_POLL_STRATEGY`
  - `PIP_DISABLE_PIP_VERSION_CHECK`
  - `PIP_PYTHON_PATH`
  - `PYTHONDONTWRITEBYTECODE`
  - `PYTHONFINDER_IGNORE_UNSUPPORTED`

Pipenv–specific environment variables:

Debug–specific environment variables:

  - `PATH`: `/usr/local/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin`
  - `LANG`: `C.UTF-8`
  - `PWD`: `/app/foo`

---------------------------

</details>

**Comment by aflopezbec** [comment]
use 	2022.10.25 version, in this version the problem not exist

**Comment by Snailedlt** [comment]
Getting the same issue. 

I got the error in #5052 first, but once I removed virtualenv with `pipenv --rm` and re-ran `pipenv shell` I got this error

**Comment by matteius** [comment]
I am not sure how the CI is passing but I will take a look at this tonight.

**Comment by matteius** [comment]
Can someone else with the issue help to verify the fix of the PR I opened #5460 
I am pretty confident that is is, but a second 👍 would be good -- and I can cut a release after that.

**Comment by Snailedlt** [comment]
> use 2022.10.25 version, in this version the problem not exist

Downgrading to this version fixed it for me too

**Comment by LukeNotable** [comment]
> Can someone else with the issue help to verify the fix of the PR I opened #5460 I am pretty confident that is is, but a second 👍 would be good -- and I can cut a release after that.

The PR version works for me

**Comment by collectiontrip** [comment]
i had same problem, you can upgrade your python 11 and error will go
